apps/web/src/routes/_content/resources/books/$bookId.tsx

```diff
@@ -76,58 +88,103 @@ function Book() {
           ) : (
             <></>
           )}
-          <Card className="mx-2 md:mx-auto">
-            <div className="my-4 w-full grid-cols-1 grid-rows-1 sm:grid-cols-3 md:grid">
-              <div className="flex flex-col items-center justify-center border-b-4 border-blue-800 md:mr-10 md:border-0">
+          <Card className="md:mx-auto" color="orange">
+            <div className="w-full flex flex-col md:flex-row">
+              <div className="flex flex-col items-center justify-center">
                 <img
-                  className="max-h-72 sm:max-h-96"
+                  className="max-w-[146px] mx-auto object-cover [overflow-clip-margin:_unset] rounded-[10px] lg:max-w-[347px] md:mx-0 lg:rounded-[22px]"
                   alt={t('imagesAlt.bookCover')}
                   src={book?.cover}
                 />
               </div>
 
-              <div className="col-span-2 my-4 flex flex-col md:mt-0">
+              <div className="w-full max-w-2xl my-4 flex flex-col md:mt-0">
                 <div>
-                  <h2 className="mb-2 max-w-lg text-2xl font-bold text-blue-800 sm:text-4xl">
+                  <h2 className="text-2xl md:text-5xl  md:font-medium text-white mb-5 lg:mb-8">
                     {book?.title}
                   </h2>
 
-                  <div className="mt-2 text-sm">
-                    <h5 className="font-light italic">
-                      {book?.author}, {book?.publicationYear}.
-                    </h5>
+                  <div className="flex flex-wrap mb-5 lg:mb-8">
+                    {book?.tags.map((object, i) => (
+                      <span
+                        key={i}
+                        className="capitalize mr-4 lg:mr-5 px-2 py-[6px] md:p-2 rounded-sm md:rounded-md text-xs font-medium lg:text-base bg-[#ffffff4d] text-white"
+                      >
+                        {object.charAt(0).toUpperCase() + object.slice(1)}
+                      </span>
+                    ))}
                   </div>
-                </div>
 
-                <div className="mt-2 text-blue-700">
-                  <span className="text-xs font-light italic">
-                    {t('book.topicsAddressed')}
-                  </span>
-                  {book?.tags.map((object, i) => (
-                    <span key={i}>
-                      {i > 0 && ', '}
-                      {object.toUpperCase()}
+                  <div className="flex">
```

Comment:
items-center to prevent this issue
![image](https://github.com/user-attachments/assets/e238daf4-c1b1-4631-bc77-b9d848ad7fbe)